app/javascript/gobierto_data/webapp/components/commons/Visualizations.vue

```diff
@@ -95,7 +105,14 @@ export default {
       return this.viewer.save()
     },
     toggleConfigPerspective() {
+      this.$root.$emit('showSavedVizString', false)
       this.viewer.toggleConfig()
+      //Enable save button when user interacts with Perspective columns
+      const itemPerspective = document.querySelector('perspective-viewer').shadowRoot
+      const rowPerspective = itemPerspective.querySelectorAll("perspective-row");
+
+      rowPerspective.forEach(rowMenu => rowMenu.addEventListener('drag', () => this.$emit("showSaving")));
+      rowPerspective.forEach(rowMenu => rowMenu.addEventListener('click', () => this.$emit("showSaving")));
```

Comment:
You could merge both actions in the same loop in order to avoid run through the array twice